packages/swapper/src/swappers/RelaySwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -0,0 +1,177 @@
+import type { Execute } from '@reservoir0x/relay-sdk'
+import type { ChainId } from '@shapeshiftoss/caip'
+import { bnOrZero, isSome } from '@shapeshiftoss/utils'
+import type { Result } from '@sniptt/monads'
+import { Err, Ok } from '@sniptt/monads'
+import type { AxiosResponse } from 'axios'
+
+import { getDefaultSlippageDecimalPercentageForSwapper } from '../../../constants'
+import type {
+  GetEvmTradeQuoteInput,
+  GetEvmTradeQuoteInputBase,
+  GetEvmTradeRateInput,
+  SwapErrorRight,
+  SwapperConfig,
+  SwapperDeps,
+  TradeQuoteStep,
+} from '../../../types'
+import { MixPanelEvent, SwapperName, TradeQuoteError } from '../../../types'
+import { makeSwapErrorRight } from '../../../utils'
+import { getRelayEvmAssetAddress } from '../utils/getRelayEvmAssetAddress'
+import { relayService } from '../utils/relayService'
+import type { QuoteParams, RelayTradeQuote, RelayTradeRate } from '../utils/types'
+
+// @TODO: implement affiliate fees
+export const getQuote = async (
+  params: QuoteParams,
+  config: SwapperConfig,
+): Promise<Result<AxiosResponse<Execute, any>, SwapErrorRight>> => {
+  return await relayService.post<Execute>(`${config.VITE_RELAY_API_URL}quote`, params)
```

Comment:
suggestion: base URL shouldn't include `/` suffix

Reply:
Changed in [4483912](https://github.com/shapeshift/web/pull/9188/commits/448391212005a43def0ab43b38186507be95ff6c)